app/views/content/train-to-be-a-teacher/teacher-training-interview.md

```diff
@@ -0,0 +1,101 @@
+
+Once the date is agreed you should recieve a confirmation email from the provider. Check this carefully to make sure you know exactly what to expect. For example, most providers ask for some type of presentation. Others may require maths or English tests to be completed in advance.
+
+## Prepare what you need to know
+
+It's a good idea to check the course provider's website to know what to expect. You can refer to this information at your interview.
+
+Consider some possible safeguarding scenarios and how you might address them. Make sure you've read the [statutory guidance for schools and colleges on safeguarding children](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2).
+
+Also, take some time to find out about [the teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) and [the national curriculum](https://www.gov.uk/government/collections/national-curriculum).
+
+## Get ready for common questions  
+
+The exact questions will vary from provider to provider but there are some common ones that you’re likely to be asked. It can help to prepare some clear answers to these in advance:
+
+* why do you want to teach?
+
+* what can you bring to the profession?
+
+* what are your interests and how could you bring these to the classroom?
+
+* what are some of the challenges faced by teachers today?
+
+## Consider what interviewers are looking for 
+
+Training providers want to see the qualities that will help you to be a good teacher. It’s worth having some examples prepared that cover common themes including:
+
+* your passion for teaching
+
+* a professional approach
+
+* how your personality comes through when presenting
+
+* confidence, energy and enthusiasm
+
+* how you're up to the challenge of teaching
+
+* an understanding of what's involved in being a teacher
+
+## Show you’ve got the right skills
+
+Training providers will also be expecting you to demonstrate you've got relevant skills, including:
+
+* how you reflect on your own performance
+
+* how you tailor information for a range of audiences
+
+* the level of your English and maths skills
+
+* the strength of your subject knowledge
+ 
+Many providers will ask you to do some form of presentation as they want to see you have the energy and passion to capture pupils' attention. Remember this is a chance to sell yourself and prove your commitment.
+
+## Prepare your own questions
+
+It’s good to have some questions of your own to ask. This helps you get more information and shows your interest in the course. You could ask:
+
+* how much time will I spend in school?  
+
+* how many schools will I get practical teaching experience at?  
+
+* what kind of schools will I be placed at? 
+
+* what is the split between exams, coursework, group work, and presentations? 
+
+* how will this course help me to get a teaching job? 
```

Comment:
This feels like quite an abrupt end - should we have a final section @philspray @EllieNodder ?

The TV interview page has a 'Stay positive' section that the end which feels nicer - something around relax, good luck, etc.?